### PR TITLE
Set main to run net7.0 until sdk+runtime are supported 

### PR DESCRIPTION
**Pull request by LoopedBard3**
Set main to run net7.0 until sdk+runtime are supported per: https://github.com/dotnet/runtime/pull/77913#issuecomment-1304220925.

@radical is this the fix you had in mind from the above comment?

### CODE REVIEW

**Comment by radical** [comment]
Honestly, I'm not sure what the *correct* fix would be, and there might not be one for now. AFAIU, we are not quite there with net8.0 project support, w.r.t the SDK, and runtime.
So, for now, reverting the earlier state would make sense.

What would be the impact of this change? Wouldn't we also need to change the tfm being passed to the command line?
It would be a good idea to also confirm that the correct assemblies, and runtime pack is being used from the workload.

cc @ericstj @carlossanlop

**Comment by radical** [comment]
Also, before merging this one do make sure to run a `dotnet-runtime-perf`+wasm build with this change.

**Comment by LoopedBard3** [comment]
Yup, got a run going atm: https://dev.azure.com/dnceng/internal/_build/results?buildId=2037521&view=results

**Comment by LoopedBard3** [comment]
Changing the main branch runs back to net7.0 and 7.0 branch will cause things to build with net7.0.

**Comment by LoopedBard3** [comment]
It looks like this fixed the wasm runs. Waiting for the rest of the test runs to finish and a review.

**Comment by LoopedBard3** [comment]
One wasm run succeeded and one had a handful of failures.

**Comment by LoopedBard3** [comment]
@radical Do you know the ETA for having net8.0 packages for wasm?

**Comment by radical** [comment]
> @radical Do you know the ETA for having net8.0 packages for wasm?

AFAIK, we are waiting for other things to be in place, like a PR that would change the default TFM in `dotnet/runtime` to `net8.0`. And we can fix anything that breaks in wasm.

**Review by caaavik-msft — APPROVED** [review]
Approving in the case that the alternative solution to use net7.0 for just wasm is not suitable

File: scripts/channel_map.py
Comment (radical) [inline scripts/channel_map.py:6]: Is this used only to decide which tfm to use on `dotnet/runtime` `main` branch?

Comment (caaavik-msft) [reply scripts/channel_map.py:6]: yes, that is correct

File: scripts/channel_map.py
Comment (caaavik-msft) [reply scripts/channel_map.py:7]: This configures what version of dotnet is installed using the dotnet installer

Comment (radical) [reply scripts/channel_map.py:7]: Ok, cool. So that doesn't affect wasm on dotnet-runtime-perf as we produce a sdk to use.

**Comment by LoopedBard3** [comment]
Closing to merge https://github.com/dotnet/performance/pull/2696 as a fix instead. This will allow wasm to build targeting net7 while the rest of the builds target net8.0.

File: scripts/channel_map.py
Comment (LoopedBard3) [reply scripts/channel_map.py:7]: Yup.

**Comment by carlossanlop** [comment]
It seems this issue is related to https://github.com/dotnet/runtime/pull/77899 . FYI, I merged it.

**Pull request closed without merging**
